src/redux/reducers.js

```diff
@@ -35,11 +31,6 @@ export const queryReducer = (state = INITIAL_STATE, action) => {
         all = all.slice(0, 20);
       }
 
-      // VISIBLE_INDEX++;
-      // setTimeout((index) => {
-      //   store.dispatch(deleteVisible(index));
```

Comment:
there is no more visible field in the store, instead i use queries.all to render the pods.